src/main.ts

```diff
@@ -190,25 +190,52 @@ const run = async (): Promise<number> => {
       info('Branch SHA', parent);
 
       // Commit files
-      const commit = await repo.commit({
-        parent,
-        branch,
-        message: render(cfg.commit.format, {
-          prefix: cfg.commit.prefix,
-          subject: render(cfg.commit.subject, {
+      let commit;
+      if (inputs.gpg_private_key !== null && inputs.gpg_passphrase !== null) {
+        commit = await repo.commit({
+          parent,
+          branch,
+          message: render(cfg.commit.format, {
+            prefix: cfg.commit.prefix,
+            subject: render(cfg.commit.subject, {
+              repository: GH_REPOSITORY,
+              index: i,
+            }),
             repository: GH_REPOSITORY,
             index: i,
           }),
-          repository: GH_REPOSITORY,
-          index: i,
-        }),
-        files: files.right.map((file) => ({
-          path: file.to,
-          mode: file.mode,
-          content: file.content,
-        })),
-        force: cfg.pull_request.force,
-      })();
+          files: files.right.map((file) => ({
+            path: file.to,
+            mode: file.mode,
+            content: file.content,
+          })),
+          force: cfg.pull_request.force,
+          name: inputs.gpg_name,
```

Comment:
To improve lisibility, for these 4 params, I believe it would be good to have the same prefix throughout the code. (i.e. all ways call it gpg_name).

For the others it's a bit more obvious, but when I first saw name above I wondered what this was.

So doing the same for all for, always calling them gpg_name, gpg_email, gpg_private_key and gpg_passphrase is probably bests.